backend/czi_hosted/default_config.py

```diff
@@ -122,6 +122,15 @@
       # If timelimit_s is set to None, then there is no time limit.
       timelimit_s: 30
 
+    metadata_cache:
+      # The maximum number of datasets that may be opened at one time.  The least recently used dataset
+      # is evicted from the cache first.
+      max_datasets: 400
```

Comment:
why 400?  they are small, right, so why not 10000 (or whatever)

Reply:
I just picked a number larger than current dataset count. But 10000 probably gives us more room for growth

Reply:
Unless there is a compelling reason/feature for a small number, pick something that we are not likely to regret over the next few years!